NameGame/NameGame/Utilities/ResponseError.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ResponseError.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/14/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+enum ResponseError: Error {
+    case emptyResponse
+    case serverError
+    case emptyImage
```

Comment:
I think `emptyResponse` adequately describes a missing image. We can remove the `emptyImage` case.